### ISSUE TITLE
GPU dropouts in abscal model

### ISSUE THREAD

**Issue by adampbeardsley** [HERA-Team/hera_cal#437]
Digging into cal_xrfi outputs, seeing gpu box dropouts that show up in abscal gains, but not in omni gains or chisq. The hunch is that these are due to things that got missed in original rfi flagging and got baked into the abscal model.
![image](https://user-images.githubusercontent.com/4752917/52087573-1541be00-2567-11e9-858c-2aff83980575.png)
We should come up with a way to feed this sort of info back into the abscal model

**Comment by jsdillon** [comment]
@nkern You might want to take a look at this. I think the issue is with the rfi flagging on the abscal models themselves.

**Comment by nkern** [comment]
Yeah this is one of the problems w calibrating to data: we could run an ag

**Comment by jsdillon** [comment]
I'm going to hijack this issue a bit to argue that it's probably not GPU dropouts, but rather RFI in the abscal model that subsequently gets spread out by the low-pass delay filter. Here's an example of this 
![image](https://user-images.githubusercontent.com/5281139/58055488-be122a00-7b12-11e9-9a28-29d2ed4b9e84.png)

Thankfully, we seem to do a pretty good job of flagging it back out, but that's not ideal (it'll lead to flags locked in LST, for example). I think we need to revisit how rfi is flagged in the abscal model.
![image](https://user-images.githubusercontent.com/5281139/58055506-cd917300-7b12-11e9-939d-755416346ed4.png)
